apps/src/applab/appStorage.js

```diff
@@ -326,7 +326,7 @@ var handlePopulateTable = function (onSuccess, onError) {
     return;
   }
 
-  if (this.status != 200) {
+  if (this.status !== 200) {
```

Comment:
I'm not sure if there's actually a risk of status being `"200"` instead of `200`.